### PR TITLE
Add mux_data pxl script for use in upcoming finagle px cli demo

### PR DESCRIPTION
**Pull request by ddelnano**
This adds a mux_data pxl script that shows mux traffic in a table. This will be used in #607 so that users can have a way to visualize the traffic deployed from the future finagle demo. The pxl script was developed as part of Twitter's hackathon in the [finagle-helloworld repo](https://github.com/SedarG/finagle-helloworld/blob/master/finagle-helloworld/misc/mux.pxl.script) to demonstrate that the mux parser was functional when it was developed earlier this year.

## Todo
- [x] Rebase once #609 is merged. This requires that the UDF in that PR exists

## Testing
- [x] Verified that `src/e2e_test/vizier/planner:planner_test` test succeeds after adding the script and fails if the mux UDF is misspelled (indicating that the mux_data pxl script is being leveraged during the test)
```
# Successful test passes

vagrant@ubuntu-jammy:/vagrant$ bazel test src/e2e_test/vizier/planner:planner_test
INFO: Analyzed target //src/e2e_test/vizier/planner:planner_test (0 packages loaded, 0 targets configured).
INFO: Found 1 test target...
Target //src/e2e_test/vizier/planner:planner_test up-to-date:
  bazel-bin/src/e2e_test/vizier/planner/planner_test_/planner_test
INFO: Elapsed time: 2.434s, Critical Path: 0.34s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
//src/e2e_test/vizier/planner:planner_test                      (cached) PASSED in 15.0s

Executed 0 out of 1 test: 1 test passes.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are
INFO: Build completed successfully, 1 total action

# Change mux UDF in pxl script to mux_req_type_name (undefined name)

vagrant@ubuntu-jammy:/vagrant$ bazel test --test_output=all src/e2e_test/vizier/planner:planner_test
--- FAIL: TestAllScriptsCompile (13.12s)
    --- FAIL: TestAllScriptsCompile/px/mux_data (0.04s)
        all_scripts_test.go:360: Planner error at 39:21  'px' object has no attribute 'mux_req_type_name'
        all_scripts_test.go:364:
FAIL
================================================================================
Target //src/e2e_test/vizier/planner:planner_test up-to-date:
  bazel-bin/src/e2e_test/vizier/planner/planner_test_/planner_test
INFO: Elapsed time: 17.031s, Critical Path: 13.22s
INFO: 2 processes: 1 internal, 1 linux-sandbox.
INFO: Build completed, 1 test FAILED, 2 total actions
//src/e2e_test/vizier/planner:planner_test                               FAILED in 13.2s
  /home/vagrant/.cache/bazel/_bazel_vagrant/6ad5fdbcbf2eaa93bd62f92333a2e6e5/execroot/px/bazel-out/k8-fastbuild/testlogs/src/e2e_test/vizier/planner/planner_test/test.log

INFO: Build completed, 1 test FAILED, 2 total actions


```

### CODE REVIEW

**Comment by ddelnano** [comment]
@htroisi this should be ready for review with the mux UDF merged.

File: src/pxl_scripts/px/mux_data/data.pxl [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''  Data Tracer
```
Comment (oazizi000) [inline src/pxl_scripts/px/mux_data/data.pxl:17]: nit: Mux Data Tracer

File: src/pxl_scripts/px/mux_data/data.pxl [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    # Order columns.
+    df = df['time_', 'source', 'destination', 'latency', 'req_name']
+
+    return df
+
+
+def add_source_dest_columns(df):
+    ''' Add source and destination columns for the mux request.
+
+    mux requests are traced server-side (trace_role==2), unless the server is
```
Comment (oazizi000) [inline src/pxl_scripts/px/mux_data/data.pxl:49]: nit: Capitalization (mux -> Mux)

Comment (ddelnano) [reply src/pxl_scripts/px/mux_data/data.pxl]: Updated.

File: src/pxl_scripts/px/mux_data/data.pxl (hunk 1 above)
Comment (ddelnano) [reply src/pxl_scripts/px/mux_data/data.pxl]: Updated.

**Review by oazizi000 — APPROVED** [review]

**Comment by htroisi** [comment]
@ddelnano two final requests on this PR and then we can merge it: 
- Could you add a `manifest.yaml` file in the `/src/pxl_scripts/px/mux_data` folder? This is used by the UI to display the script description. 
- Please run `/src/pxl_scripts/update_readme.py` to update the list of scripts in the README file.

**Review by htroisi — APPROVED** [review]

**Review by oazizi000 — APPROVED** [review]

**Pull request closed without merging**